crates/pallet-domains/src/lib.rs

```diff
@@ -458,6 +475,10 @@ mod pallet {
         /// before requesting new withdrawal.
         #[pallet::constant]
         type WithdrawalLimit: Get<u32>;
+
+        /// Current bundle version accepted by the runtime.
+        #[pallet::constant]
+        type CurrentBundleAndExecutionReceiptVersion: Get<BundleAndExecutionReceiptVersion>;
```

Comment:
This may require a fraud proof.

For example, the bundle version is changed at domain block  `#10`, the domain chain continues progressing until block `#15`, and then a fraud proof submitted prune the ER of block `#5`. Now the honest operator needs to re-submit the V0 ER of block `#5..#9` and the V1 ER of `#10..#15`, if a malicious operator preemptively submit a wrong version ER e.g. V1 ER of block `#5`, then the honest operator will need a fraud proof to prune this ER.

An alternative is to keep track of the ER version of a given domain block in the runtime state, so the consensus chain can check and simply reject the wrong version ER.